### DISCUSSION TITLE
Can i add more than 5 menu items in bottom navigation view?

### DISCUSSION THREAD

**Discussion by adil-qt** [community/community#130663] [Mobile] [answered]
### Select Topic Area

Question

### Body

I am trying to add more then 5 items in bottom navigation view but its causing inflate error.

**Comment by aiyu-ayaan** [comment, marked as answer]
If you want that, u can take a Horizontal linear layout in that you take the horizontal scroll and then take the text view or button as much as u like.

**Reply by Gececi665** [reply]
Teşekkürler
